examples/relayToSystem.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * When importing from @substrate/asset-transfer-api it would look like the following
+ *
+ * import { AssetsTransferApi, constructApiPromise } from '@substrate/asset-transfer-api'
+ */
```

Comment:
can we add as a comment a short description of the example? For example, in this case it would be something like:

"In this example we are creating a call to send X amount of WND from Westend relaychain to a user account in parachain Y and with a limit in execution weight at the destination".

Reply:
Updated